client/cmd/admin/create-conn.go

```diff
@@ -179,6 +194,30 @@ var createConnectionCmd = &cobra.Command{
 	},
 }
 
+func verifyAccessModeStatus(mode string) string {
+	if slices.Contains(connAccessModesFlag, mode) {
+		return "enabled"
+	} else if !slices.Contains(connAccessModesFlag, mode) {
+		return "disabled"
+	}
+
+	styles.PrintErrorAndExit("invalid access mode: %q", mode)
+	return "disabled"
+}
+
+func verifySchemaStatus(schema string, connType string) string {
+	if schema == "" && connType == "database" {
+		return "enabled"
+	} else if schema == "" {
+		return "disabled"
+	} else if schema == "enabled" || schema == "disabled" {
+		return schema
+	}
+
+	styles.PrintErrorAndExit("invalid schema status: %q", schema)
+	return "disabled"
```

Comment:
- Inform the user the valid options instead of just informing that something is incorrect.
Example: "invalid value for schema status %s, accepted values are: [enabled disabled]"

- The `return "disabled'` will never be returned, return an empty string instead